content/questions/question-060.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Question 060"
+date: 2023-09-17T20:47:56Z
+draft: false
+subject: []
+---
+
+# Which of the following default environment variables contains the name of the person or app that initiated the workflow run?
+
+> https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables
+
+1. [ ] `GITHUB_USER`
+1. [ ] `GITHUB_REPOSITORY`
+1. [ ] `GITHUB_WORKFLOW`
+1. [x] `GITHUB_ACTOR`
+> The `GITHUB_ACTOR` environment variable contains the name of the person or app that initiated the workflow run. For example, `octocat`.
```

Comment:
These explanations are displayed when someone picks an answer that is both:
- wrong
- has an explanation

This explanation has no effect as this is the correct answer. So You can remove it.

I will remember to update that in [CONTRIBUTING](https://github.com/FidelusAleksander/githubcertified/blob/master/CONTRIBUTING.md) guide

Reply:
Thanks, I've removed the explanation from the question